examples/class_imbalance/credit_fraud.py

```diff
@@ -0,0 +1,71 @@
+fraud_inputs = [
+    {"name": "Time", "type": "numerical", "output_flag": False},
```

Comment:
Remove the `output_flag` from these input maps.

Also, where/how is this file used?

Reply:
Having this file in here is a mistake, it's actually a file that was going to be used in the example with the credit card fraud dataset. I accidentally committed it so I can remove in the next commit.